tasties_app/tests/test_recipes_view.py

```diff
@@ -11,3 +11,14 @@ def test_recipes_view(self, client, signed_up_credentials):
         response = client.post('/login/', data={'username': VALID_USER, 'password': VALID_PASSWORD})
         assert response.status_code == 302
         assert response.url == '/'
+
+    def test_view_recipe(self, client, signed_up_credentials):
+        client.post('/login/', data={'username': VALID_USER, 'password': VALID_PASSWORD})
+        response = client.get('/view_recipe/')
+        assert response.status_code == 302
```

Comment:
Just want to validate, another test to check status code 200 is going to be part of another PR?

Reply:
Yes, because at the moment it's not accessible from the app.
But in the next PR, I will add a test that shows a successful  HTTP response.

Reply:
Great, thanks!